client/keys/add_ledger_test.go

```diff
@@ -72,7 +72,7 @@ func Test_runAddCmdLedgerWithCustomCoinType(t *testing.T) {
 	require.Equal(t, "keyname1", key1.GetName())
 	require.Equal(t, keyring.TypeLedger, key1.GetType())
 	require.Equal(t,
-		"terrapub1addwnpepqvpg7r26nl2pvqqern00m6s9uaax3hauu2rzg8qpjzq9hy6xve7sw0d84m6",
+		"terrapub1addwnpc2yyps9rcdt20ag9sqrywdal02qhnh56xlhn3gvfquqxggqkungen86pcn2nzzw",
```

Comment:
Do you know why these addresses changed?

Reply:
This is concerning...If we have an address that exists on the Hub now, will it change in stargate?

Reply:
I believe this is due to the fact that pub keys are now defined as struct and it's breaking amino serialization, overriding Amino marshaling should fix that cf. https://github.com/cosmos/cosmos-sdk/pull/7276, thoughts @amaurymartiny? Reverting back to the old values.

Reply:
Yes, you're right, adding amino marshaling fixed all tests.